### PR TITLE
Pre-commit: remove `aiida_quantumespresso.calculations` from exclude

### PR DESCRIPTION
**Pull request by sphuber**

### CODE REVIEW

File: aiida_quantumespresso/calculations/helpers/__init__.py [hunk 1]
```diff
@@ -149,164 +139,166 @@ def pw_input_helper(input_params, structure,
...
-        versions = sorted(versions, key=lambda x: StrictVersion(x))
+        versions = [
+            fname[len(prefix):-len(suffix)]
+            for fname in os.listdir(module_dir)
+            if fname.startswith(prefix) and fname.endswith(suffix)
+        ]
+        versions = sorted(versions, key=lambda x: Version(x))  # pylint: disable=unnecessary-lambda
         strictversions = versions + [version]
-        strictversions = sorted(strictversions, key=lambda x: StrictVersion(x))
+        strictversions = sorted(strictversions, key=lambda x: Version(x))  # pylint: disable=unnecessary-lambda
```
Comment (greschd) [inline aiida_quantumespresso/calculations/helpers/__init__.py]: I think here pylint might be right: Can't you just put ``key=Version``?

Comment (sphuber) [reply aiida_quantumespresso/calculations/helpers/__init__.py]: Ah yes of course, I forgot that key just takes any callable

Comment (sphuber) [reply aiida_quantumespresso/calculations/helpers/__init__.py]: I fixed it. If you could approve again, I can merge it, thanks!

Comment (greschd) [reply aiida_quantumespresso/calculations/helpers/__init__.py]: Done! Not sure it helps, the approval needs to be "from someone with write access". But I'm guessing you have the power to merge anyway?

**Pull request merged**